exist-core/src/test/xquery/xquery3/fnElementWithId.xqm

```diff
@@ -0,0 +1,50 @@
+(:
+ : eXist-db Open Source Native XML Database
+ : Copyright (C) 2001 The eXist-db Authors
+ :
+ : info@exist-db.org
+ : http://www.exist-db.org
+ :
+ : This library is free software; you can redistribute it and/or
+ : modify it under the terms of the GNU Lesser General Public
+ : License as published by the Free Software Foundation; either
+ : version 2.1 of the License, or (at your option) any later version.
+ :
+ : This library is distributed in the hope that it will be useful,
+ : but WITHOUT ANY WARRANTY; without even the implied warranty of
+ : MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ : Lesser General Public License for more details.
+ :
+ : You should have received a copy of the GNU Lesser General Public
+ : License along with this library; if not, write to the Free Software
+ : Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ :)
+xquery version "3.1";
+
+module namespace fnewi="http://exist-db.org/xquery/test/function_element_with_id";
+
+declare namespace test="http://exist-db.org/xquery/xqsuite";
+declare namespace x="httpx://x/ns/1.0";
```

Comment:
This namespace declaration doesn't appear to be used.

Reply:
Thank you for catching this.  I removed the unused namespace declaration.